chapters/equations.tex

```diff
@@ -311,6 +312,45 @@ \subsubsection{Restrictions on Equations within When-Equations}\label{restrictio
 value of \lstinline!y! from the previous event instant.
 \end{nonnormative}
 
+\begin{example}
+The restrictions for if-equations mean that both of the following variants are illegal:
+\begin{lstlisting}[language=modelica]
+  Real x, y;
+equation
+  if time < 1 then
+    when sample(1,2) then
```

Comment:
Is this according to a general rule about not having space after comma, or is it more of a special case?  Whatever the answer is, it's the kind of thing that I think we should spell out in a style guide, so that the entire document can converge to the same style everywhere.

Reply:
(I tried to reverse-engineer Dymola by looking at the MSL, but the closest thing to a rule that I could find was that whitespace was generally skipped inside annotations, and often present elsewhere.)

Reply:
I changed to space after comma here.
But just to be clear: Dymola's default whitespace rules and mine aren't identical.

Reply:
I see.  Resolving.